config.toml

```
@@ -1,3 +1,131 @@
 baseURL = "https://unicef.github.io/drone-dpgtoolkit"
 languageCode = "en-us"
 title = "Drone DPG Toolkit"
+theme = "inventory"
+enableEmoji = true
+enableGitInfo = true
+
+# Google Analytics
+googleAnalytics = "G-80FMX4TPM0"
```

This is the Google Analytics code for the Open Source Inventory. We don't want to include this in the Drone DPG Toolkit since it will break the page analytics for the O.S. Inventory.

```suggestion
googleAnalytics = ""
```